webapp/src/viewModel/boardTree.ts

```diff
@@ -55,57 +55,54 @@ class MutableBoardTree implements BoardTree {
     allCards: MutableCard[] = []
     rawBlocks: IBlock[] = []
 
-    workspaceUsers: WorkspaceUsers = {
-        users: new Array<IUser>(),
-        usersById: new Map<string, IUser>(),
-    }
+    usersById: {[key: string]: IUser}
 
     get allBlocks(): IBlock[] {
         return [this.board, ...this.views, ...this.allCards, ...this.cardTemplates, ...this.rawBlocks]
     }
 
-    constructor(board: MutableBoard) {
+    constructor(board: MutableBoard, usersById: {[key: string]: IUser}) {
         this.board = board
+        this.usersById = usersById
```

Comment:
Should this use `setUsersById(usersById)` instead of a direct assign?

Reply:
Not very important, actually that new function is not used anywhere, so I can remove the function.